tests/test_roman.py

```diff
@@ -322,7 +322,7 @@ def test_roman_backgrounds():
     # The routine should not allow us to look directly at the sun since the background there is high
     # (to understate the problem).  If no date is supplied, then the routine assumes RA=dec=0 means
     # we are looking at the sun.
-    bp_dict = galsim.roman.getBandpasses()
+    bp_dict = galsim.roman.getBandpasses(include_all_bands=True)
```

Comment:
I know you normally like to keep the old tests as they were, but I _think_ this should be fine?

Reply:
This change shouldn't be necessary.  However, you should add a test that the Prism/Grism bands aren't included yet.  Then redo with `include_all_bands=True` and check that they are included.